pkg/service/tenant.go

```diff
@@ -52,26 +52,32 @@ func (s *tenantService) Update(ctx context.Context, req *v1.TenantUpdateRequest)
 
 func (s *tenantService) Delete(ctx context.Context, req *v1.TenantDeleteRequest) (*v1.TenantResponse, error) {
 	tenant := req.NewTenant()
-	tenantFilter := map[string]any{
+	hostFilter := map[string]any{
```

Comment:
for me hostfilter is hard to understand, maybe rootFilter ?

Reply:
How about `tenantIsHostFilter` and `tenantIsMemberFilter`?